<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleContactWidget</class>
 <widget class="QWidget" name="SingleContactWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40, 45, 66);
font: 12px &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="remarkLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(24,140,249);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QToolButton" name="transferBtn">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>80</y>
     <width>112</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{color: rgb(70,82,113);border:1px solid rgb(70,82,113);border-radius:15px;background-color:white;background-origin: content;background-image: url(&quot;:/ui/wallet_ui/contactTransfer.png&quot;);background-repeat: no-repeat; background-position: center left;padding:0px 0px 0px 15px;}
QToolButton:hover{color: white;border:1px solid rgb(24,140,249);background-color:rgb(24,140,249);background-image: url(&quot;:/ui/wallet_ui/contactTransfer_hover.png&quot;);}</string>
   </property>
   <property name="text">
    <string>TRANSFER</string>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>12</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-image:url(:/ui/wallet_ui/contactEditBtn.png);background-repeat: no-repeat;background-position: center;border-style: flat;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
